frontend/src/pages/TermsOfUsePage/TermsOfUsePage.tsx

```diff
@@ -1,163 +1,109 @@
-              to another site, you are subject to the privacy policy of the new
-              web site.
-            </p>
-          </li>
-          <li className='pb-5'>
-            <h3 className='font-medium font-sansTitle text-title'>
-              Changes to our Privacy Policy
-            </h3>
-            <p className='font-sansText text-small'>
-              We may change the terms and conditions of our Privacy Policy at
-              any time by posting revisions on the MSM and HET web site. By
-              accessing or using the MSM and HET web site, you agree to be bound
-              by all the terms and conditions of our Privacy Policy as posted on
-              the MSM and HET web site at the time of your access or use. If you
-              do not agree to the terms of this Privacy Policy or any revised
-              statement, please exit the site immediately.
-            </p>
-          </li>
-          <li className='pb-5'>
-            <h3 className='font-medium font-sansTitle text-title'>
-              Complaint Process
-            </h3>
-            <p className='font-sansText text-small'>
-              If you have a complaint or problem with the HET website, or if you
-              believe your privacy rights have been violated from the HET
-              website, you may email us at HET@msm.edu. Please indicate the
-              reason for contacting us. The HET Communications and Dissemination
-              Core will review your complaint for response or resolution.
-            </p>
-          </li>
-          <li className='pb-5'>
-            <h3 className='font-medium font-sansTitle text-title'>
-              Disclaimer
-            </h3>
-            <p className='font-sansText text-small'>
-              No data protection method or combination of methods can be
-              guaranteed as completely secure. MSM nor HET are responsible for
-              and will not be held liable for disclosures of your personal
-              information due to errors in transmissions or unauthorized acts of
-              third parties. MSM nor HET guarantee the privacy of your
-              confidential information transmitted to its web site should you
-              choose not to use the appropriate secure on-line forms provided in
-              the relevant pages of the web site. By using this web site you
-              agree to the terms and conditions outlined in this Privacy Policy
-              statement.
-            </p>
-          </li>
+      <section
+        id='main-content'
+        aria-labelledby='main-content'
+        className='mx-auto flex w-svw max-w-lgXl flex-col justify-center px-8 py-16'
+      >
+        <h1
+          id='main'
+          className='font-bold font-sansTitle text-altGreen text-bigHeader leading-lhNormal'
+        >
+          Terms of Use
+        </h1>
+        <h2 className='sr-only'>Terms of Use</h2>
```

Comment:
This is duplicate of the h1 which is visible. Should be removed and the subsequent section headers should be h2 i think